### ISSUE TITLE
[BUG] Adding a new Rarity or feat doesnt do anything

### ISSUE THREAD

**Issue by Dyrnwyrm** [p4535992/foundryvtt-rarity-colors#18]
**Module Version:** v0.3.19

**Before open any issue**

1) Go to rarity colors configuration
2) Add a new type of rarity with a color (test for this example)
3) Reload the app for the changes to apply
4) The new rarity is not present at all, same with the feat types

I tried on a clean world with only rarity colors + lib colors enabled and the rarity nor the feature added the new categories, maybe dnd changed it around so the properties are not there for adding the new stuff.
 
 
 Dnd: 3.1.2
 Foundry: 11.315
 
Screenshots:
 
![image](https://github.com/p4535992/foundryvtt-rarity-colors/assets/129613553/5f8fa7cb-ae3f-4af7-b078-f1c99266e346)
![image](https://github.com/p4535992/foundryvtt-rarity-colors/assets/129613553/6f6409eb-372e-46d5-a80d-92cf3a86bba2)

**Comment by p4535992** [comment]
That is correct the module is only designed to associate a color with an item rarity already present not to create it. 
If you have to create a new rarity you have to use the standard mechanisms of the dnd5e system or other modules like https://theripper93.com/module/dnd5e-system-customizer or https://foundryvtt.com/packages/custom-dnd5e
